apps/wrf/readme.md

```diff
@@ -1,110 +1,461 @@
+
+- Test 1 
+  - SKU: Standard_HB120rs_v2
+  - Nodes: 1 
+  - Processes per node: 60 
+  - MPI processes per node: 60 
 ```
-./ungrib.exe >& ungrib.log
+mkdir ~/test1
+cd ~/test1
+
+qsub -l select=1:nodearray=execute1:ncpus=60:mpiprocs=60,place=scatter:excl -v "SKU_TYPE=hbv2,INPUTDIR=/apps/hbv2/wrf-openmpi/WRF-4.1.5/run" /data/azurehpc/apps/wrf/run_wrf_openmpi.pbs
+```
+
+- Test 2 
+  - SKU: Standard_HB120rs_v2
+  - Nodes: **2** 
+  - Processes per node: 60 
+  - MPI processes per node: 60 
 ```
+mkdir ~/test2
+cd ~/test2
+
+qsub -l select=2:nodearray=execute1:ncpus=60:mpiprocs=60,place=scatter:excl -v "SKU_TYPE=hbv2,INPUTDIR=/apps/hbv2/wrf-openmpi/WRF-4.1.5/run" /data/azurehpc/apps/wrf/run_wrf_openmpi.pbs
 ```
-mpirun ./metgrid.exe
+
+- Test 3 
+  - SKU: Standard_HB120rs_v2
+  - Nodes: **3** 
+  - Processes per node: 60 
+  - MPI processes per node: 60 
 ```
-Should see the following files
+mkdir ~/test3
+cd ~/test3
+
+qsub -l select=3:nodearray=execute1:ncpus=60:mpiprocs=60,place=scatter:excl -v "SKU_TYPE=hbv2,INPUTDIR=/apps/hbv2/wrf-openmpi/WRF-4.1.5/run" /data/azurehpc/apps/wrf/run_wrf_openmpi.pbs
 ```
--rw-rw-r--. 1 hpcuser hpcuser  79075286 Dec 12 04:11 met_em.d01.2018-06-17_12:00:00.nc
--rw-rw-r--. 1 hpcuser hpcuser  78622472 Dec 12 04:11 met_em.d01.2018-06-17_09:00:00.nc
--rw-rw-r--. 1 hpcuser hpcuser  79054700 Dec 12 04:10 met_em.d01.2018-06-17_06:00:00.nc
--rw-rw-r--. 1 hpcuser hpcuser  78668607 Dec 12 04:10 met_em.d01.2018-06-17_03:00:00.nc
--rw-rw-r--. 1 hpcuser hpcuser  79435709 Dec 12 04:10 met_em.d01.2018-06-17_00:00:00.nc
+
+- Test 4
+  - SKU: Standard_HB120rs_v2
+  - Nodes: **4** 
+  - Processes per node: 60 
+  - MPI processes per node: 60 
 ```
-cd WRF v4 run directory
-cp new_conus2.5km namelist.input to this location
-cp the file met_em*.nc files to this location
+mkdir ~/test4
+cd ~/test4
 
+qsub -l select=4:nodearray=execute1:ncpus=60:mpiprocs=60,place=scatter:excl -v "SKU_TYPE=hbv2,INPUTDIR=/apps/hbv2/wrf-openmpi/WRF-4.1.5/run" /data/azurehpc/apps/wrf/run_wrf_openmpi.pbs
 ```
-mpirun ./real.exe
+
+- Test 5
+  - SKU: Standard_**HB120rs_v3**
+  - Nodes: **3** 
+  - Processes per node: 60 
+  - MPI processes per node: 60
 ```
-The following files should be generated (These are the input files required to run WRF v4)
+mkdir ~/test5
+cd ~/test5
+
+qsub -l select=3:nodearray=execute1:ncpus=60:mpiprocs=60,place=scatter:excl -v "SKU_TYPE=hbv2,INPUTDIR=/apps/hbv2/wrf-openmpi/WRF-4.1.5/run" /data/azurehpc/apps/wrf/run_wrf_openmpi.pbs
 ```
--rw-rw-r--. 1 hpcuser hpcuser  31285895 Dec 12 15:50 wrfbdy_d01
--rw-rw-r--. 1 hpcuser hpcuser 127618030 Dec 12 15:50 wrfinput_d01
+
+- Test 6
+  - SKU: Standard_**HB120-64rs_v3**
+  - Nodes: 3 
+  - Processes per node: **64** 
+  - MPI processes per node: **64** 
+```
+mkdir ~/test6
+cd ~/test6
+
+qsub -l select=3:nodearray=execute1:ncpus=64:mpiprocs=64,place=scatter:excl -v "SKU_TYPE=hbv2,INPUTDIR=/apps/hbv2/wrf-openmpi/WRF-4.1.5/run" /data/azurehpc/apps/wrf/run_wrf_openmpi.pbs
```

Comment:
HB120-64rs_v3 test, but hbv2 references?

Reply:
Same above. During my tests, I've used hbv2 reference and I was able to perform tests successfully on HBv2 and HBv3.
Do you recommend changing to hbv3 reference when running on HBv3?
If I change it, do I need to run the WRF and WPS build again?

Reply:
I think it's good practice to build on the SKU you are running on. It's confusing to run on HBv3 but reference hbv2. To simply to documentation, I would just pick HBv3 (because its newer than hbv2) and give a few examples running specifically on HBv3. You could then add a note to state that a very similar procedure can be also used to run WRF on hbv2.